prov/efa/src/rxr/rxr_pkt_type_rts.c

```diff
@@ -517,6 +519,13 @@ void rxr_pkt_proc_shm_long_msg_rts(struct rxr_ep *ep, struct rxr_rx_entry *rx_en
 	struct iovec *iovec_ptr;
 	int ret, i;
 
+	/*
+	 * rx_entry->cq_entry.len is the buffer size, rx_entry->total_len
+	 * is incoming data size.
+	 */
+	if (rx_entry->cq_entry.len > rx_entry->total_len)
```

Comment:
This looks like this is a separate bug - can you move this to a separate commit so we can fix this in the next 1.9.x release as well?

Reply:
Thank you for pointing out. This is part of a future commit, mixed in here during rebase. Current branch sets rx_entry->cq_entry.len correctly in rxr_pkt_handle_rma_context_send_completion(). No need to change.

I removed this piece in new revision.